source/DataBase/DataBaseCommon.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <vector>
+#include <string>
+struct DataBaseCommon
+{
+    static const std::vector<std::string> userRecordsRecords;
+    static const std::vector<std::string> categoriesRecords;
+    static const std::vector<std::string> passwordHistoryRecords;
+    static const std::vector<std::string> passwordRecords;
+
+    static const std::string dataBasePath;
+
+    static const std::string users;
+    static const std::string passwords;
+    static const std::string passwordHistory;
+    static const std::string categories;
```

Comment:
I believe the functionality of a string isn't necessary in this context. Since we're simply passing a constant string to the SQLite function and subsequently calling `c_str()`, it's more appropriate and efficient to utilize a plain `char[]` instead.
```
namespace database
{
    namespace common
    {
        static constexpr char dataBasePath[] = "../../../source/dataBase/password-manager.db";
    }
}
```